cdk/package.json

```diff
@@ -9,23 +9,28 @@
     "watch": "tsc -w",
     "test": "jest",
     "cdk": "cdk",
-    "synth": "cdk synth rule-manager > ./cfn/rule-manager.cfn.yaml && cdk synth rule-manager-db > ./cfn/rule-manager-db.cfn.yaml"
+    "synth": "cdk synth"
```

Comment:
Would suggest [adding](https://github.com/guardian/cdk-cli/blob/dd3460d17937a9de3c24bfe11f8bad0bba0f66e5/src/template/package.json#L16) `--path-metadata false --version-reporting false` flags to the `synth` command to reduce bloat in the resulting CFN template.